src/pages/workspace/WorkspacePageWithSections.tsx

```diff
@@ -130,7 +130,8 @@ function WorkspacePageWithSections({
     }, [shouldSkipVBBACall]);
 
     const shouldShow = useMemo(() => {
-        if (isEmptyObject(policy) && isEmptyObject(policyDraft)) {
+        // If the policy object doesn't exists or contains only errors, we shouldn't display it.
```

Comment:
what does "policy only having errors" indicate?

Reply:
I've refactored this comment a bit. It means that the policy object has only one property `errors` which stores information about the type of error that occurred, e.g. `Invalid Policy ID`. In this case, we want to skip these objects.

Reply:
okay makes sense thank you for the explanation 🙇